thread/src/main/kotlin/io/github/paulgriffith/kindling/thread/ThreadComparisonPane.kt

```diff
@@ -85,7 +91,28 @@ class ThreadComparisonPane(
         threads.firstOrNull { it != null }?.let {
             header.setText(it)
         }
+
+        val onlyOneThread = threads.filterNotNull().size == 1
+
+        val highestCpu = if (!onlyOneThread) {
+            threads.groupBy {
+                it?.cpuUsage ?: 0.0
+            }.maxByOrNull { it.key }?.takeIf {
+                it.value.size == 1
+            }?.key
```

Comment:
I would be surprised if there weren't a cleaner way to express this, but I'll have to put it in the IDE to check